common/source/docs/common-downloading-and-analyzing-data-logs-in-mission-planner.rst

```diff
@@ -358,6 +358,30 @@ but the most common are:
 | 25         | Set Home (home location coordinates have been capture)                                             |
 +------------+----------------------------------------------------------------------------------------------------+
 
+
+**GPA: (Global Position Accuracy)**
+
++------------+----------------------------------------------------------------------------------------------------+
+| FIELD      | DESCRIPTION                                                                                        |
++------------+----------------------------------------------------------------------------------------------------+
+| TimeUS     | The GPS reported time since epoch in milliseconds                                                  |
```

Comment:
This isn't accurate (it's in microseconds not milliseconds and it's the timestamp of when the dataflash message was written, rather then being associated with any GPS data), it's also not listed for any other message in this table, I'd drop it from the list.